x/sequencer/keeper/invariants.go

```diff
@@ -95,3 +97,54 @@ func ProposerBondedInvariant(k Keeper) sdk.Invariant {
 		), broken
 	}
 }
+
+// SequencerMinBondInvariant checks if the sequencer always maintains the minimum bond as long as it is bonded status
+func SequencerMinBondInvariant(k Keeper) sdk.Invariant {
```

Comment:
hmm. pretty sure that's not an agreed upon invariant (for this stage).
if someone puts minimum bond, and than gets first slash for inactivity this invariant will break.